src/index.ts

```diff
@@ -236,13 +236,10 @@ function main() {
                         answers.dockerTag = 'testing';
                     } else {
                         // For a released version the docker tag and version should be same
-                        // Default to master and then try to get the version number from package.json
-                        answers.verion = 'master';
-                        const list = RELEASE_VERSION_PATTERN.exec(packageJson.version);
-                        if (list) {
-                            answers.version = list[1];
-                        }
-                        answers.dockerTag = answers.version;
+                        // Default to latest released verion
+                        const version = '1.0.0';
```

Comment:
should this be 1.0.1 like in package.json?

Reply:
The package.json version is different then our released version for RMv2